README.md

```diff
@@ -1159,6 +1159,20 @@ Vagrant.configure(2) do |config|
 end
 ```
 
+## Custom command line arguments
+You can also specify multiple commandline arguments for qemu-system
+
+* `value` - Value
+
+```ruby
+Vagrant.configure("2") do |config|
+  config.vm.provider :libvirt do |libvirt|
+    libvirt.commandline :value => "-device"
+    libvirt.commandline :value => "isa-applesmc,osk=ourhardworkbythesewordsguardedpleasedontsteal(c)AppleComputerInc"
+  end
+end
+```
+
```

Comment:
I though that this was enough, am I missing something?

```
      def commandline(options = {})
        @commandline_args = [] if @commandline_args == UNSET_VALUE
        @commandline_args.push(value: options[:value])
      end
```

Besides I checked it (and iff I packaged it right), it works.